File: tri/tri.py
#!/usr/bin/env python
# coding: utf-8

import pandas as pd

# import des données
data = pd.read_csv("data.csv",sep=";")

# tri à plat
data["csp"].value_counts()

# tri à plat avec probabilités
data["csp"].value_counts(normalize=True,dropna=False)

# tri croisé version pivot_table()
data.pivot_table(index="csp",columns="ville",aggfunc=len)

# tri croisé version crosstab
pd.crosstab(data["csp"],data["ville"])

File: tri/tri.r
# récupération des données
data <- read.csv2("data.csv",fileEncoding="UTF-8-BOM")

# tri à plat
table(data$csp)

summary(data$csp)

install.packages("questionr",repos='http://cran.us.r-project.org')
library(questionr)

freq(data$csp)

# tri croisé
table(data$csp,data$ville)

install.packages("gmodels",repos='http://cran.us.r-project.org')
library(gmodels)

CrossTable(data$csp,data$ville)